src/front/js/component/footer.js

```diff
@@ -1,10 +1,45 @@
-import React, { Component } from "react";
+import React from "react";
+import { Link } from "react-router-dom";
 
-export const Footer = () => (
-	<footer className="footer mt-auto py-3 text-center">
-		<p>
-			Made with <i className="fa fa-heart text-danger" /> by{" "}
-			<a href="http://www.4geeksacademy.com">4Geeks Academy</a>
-		</p>
-	</footer>
-);
+export const Footer = () => {
+  return (
+    <>
+      <footer className="page-footer font-small blue pt-4">
+        <div id="links" class="container text-center">
+          <div class="row row-cols-1 row-cols-sm-2 row-cols-md-4">
+            <div>
+              <Link to="/">
+                <h6>Ask to the Community</h6>
+              </Link>
+            </div>
+            <div>
+              <Link to="/">
+                <h6>FAQ's</h6>
+              </Link>
+            </div>
+            <div>
+              <Link to="/">
+                <h6>Contact Us</h6>
+              </Link>
+            </div>
+            <div>
+              <Link to="/">
+                <h6>About Us</h6>
+              </Link>
+            </div>
+          </div>
+        </div>
+        <div id="icons-container" class="container md">
+          <div id="icons" className="container-fluid text-center">
+            <i id="twitter" class="fa fa-brands fa-twitter"></i>
+            <i class="fa fa-instagram" aria-hidden="true"></i>
+            <i class="fa fa-facebook-official" aria-hidden="true"></i>
```

Comment:
Could you add some hyperlinks to these icons? It can be even an empty path, but just so it looks like a link, you know?

It feels weird to hover over them and not to see the cursor turns into that pointer hahah

Oh! Another thing...! You probably forgot to change those "class" attributes to "className", hm?
It happens to me all the time, no worries! Always check your console tab, it's the best way to find these bad boys 😎